contracts/script/DeployFeeContract.s.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+
+import { FeeContract } from "../src/FeeContract.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract DeployFeeContract is Script {
+    function run() external returns (address payable proxy, address admin) {
+        // string memory seedPhrase = vm.envString("MNEMONIC");
+        // (address admin,) = deriveRememberKey(seedPhrase, 0);
+        (proxy, admin) = deployFeeContract();
```

Comment:
well actually when I deployed to testnet, i passed the private key in as an env variable, however, would we use a multisig for the actual deployment?

Reply:
if we are using multisig, we should also be loading the multisig account/address from env var, right?

if so, here loading from `envString` is closer to actual code?

Reply:
So it depends on how we manage it. for example, if we use Openzeppelin Defender for managing the deployment of the contracts the script would look like this https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades/blob/main/test/Defender.s.sol (though at the moment this is not sufficient since deployments with defender via Foundry doesn't support constructor arguments, but I think Hardhat + Defender does). So I can uncomment this code for now but it's likely to change.

Reply:
thanks for the info! we can leave it as is for now

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/65917037d9433e95af5fe4c54c7060535b7ebe74